js_modules/dagit/packages/core/src/assets/AssetPartitionDetail.tsx

```diff
@@ -218,6 +219,10 @@ export const AssetPartitionDetail: React.FC<{
         <Subheading>Source Data</Subheading>
         <AssetMaterializationUpstreamData timestamp={latest?.timestamp} assetKey={assetKey} />
       </Box>
+      <Box padding={{top: 24}} flex={{direction: 'column', gap: 8}}>
+        <Subheading>System Tags</Subheading>
```

Comment:
`System tags`

Reply:
Will fix this on this whole page - looks like the casing is all title case eg "Source Data"